src/config.rs

```diff
@@ -0,0 +1,360 @@
+use crate::{Result, Target, Toml};
+
+use std::collections::HashMap;
+use std::env::var;
```

Comment:
```suggestion
use std::env;
```

Calling it as `env::var` is a bit more readable I think.